solidity/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keep-network/tbtc",
-  "version": "1.1.2-pre",
+  "version": "1.1.2-rc",
```

Comment:
This doesn't match `package.json`.

Reply:
Fixed in 3b66897658d77aa9b095176442a534e510a9a7a1.